README.md

```diff
@@ -218,6 +219,21 @@ test('it works', () => {
 })
 ```
 
+## react-testing-library
+
+To generate snapshots with [react-testing-library](https://github.com/kentcdodds/react-testing-library), you can just follow the example below:
+
+```js
+import { render } from 'react-testing-library'
+
+test('it works', () => {
+  const { container } = render(<Button />)
+  expect(container).toMatchSnapshot()
```

Comment:
I would change this to `container.firstChild` (as per `react-testing-library` [documentation](https://github.com/kentcdodds/react-testing-library#usage)).